PowerValidatedSolutions.psm1

```diff
@@ -32750,18 +32956,47 @@ Function Test-vRAAuthentication {
 Export-ModuleMember -Function Test-vRAAuthentication
 
 Function Test-WSAConnection {
+    <#
+        .SYNOPSIS
+        Check network connectivity to a Workspace ONE Access cluster or node.
+
+        .DESCRIPTION
+        Checks the network connectivity to a Workspace ONE Access cluster or node.
+        Supports testing a connection on ports 443 (HTTPS), 8443 (HTTPS), 22 (SSH). Default: 443 (HTTPS).
+
+        .EXAMPLE
+        Test-WSAConnection -server xint-vrops01.rainpole.io
+        This example checks network connectivity with a Workspace ONE Access cluster or node on default port, 443 (HTTPS).
+
+        .EXAMPLE
+        Test-WSAConnection -server xint-vrops01.rainpole.io -port 443
+        This example checks network connectivity with a Workspace ONE Access cluster or node on port 443 (HTTPS). This is the default port.
+
+        .EXAMPLE
+        Test-WSAConnection -server xint-vrops01a.rainpole.io -port 22
```

Comment:
```suggestion
        Test-WSAConnection -server xint-wsa01.rainpole.io -port 22
```

Reply:
Addressed in 14b2e73bdb02f296b8ecbcda1032260e254946ce.